extensions/avro-extensions/src/main/java/io/druid/data/input/guice/AvroParsersModule.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Druid - a distributed column store.
+ * Copyright 2012 - 2015 Metamarkets Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.druid.data.input.guice;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.jsontype.NamedType;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.google.inject.Binder;
+import io.druid.data.input.AvroHadoopInputRowParser;
+import io.druid.data.input.AvroParseSpec;
+import io.druid.data.input.AvroStreamInputRowParser;
+import io.druid.initialization.DruidModule;
+import org.schemarepo.Repository;
+import org.schemarepo.client.RESTRepositoryClient;
+import org.schemarepo.config.Config;
+import org.schemarepo.json.GsonJsonUtil;
+import org.schemarepo.json.JsonUtil;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class AvroParsersModule implements DruidModule
+{
+  public AvroParsersModule() {}
+
+  @Override
+  public List<? extends Module> getJacksonModules()
+  {
+    return Arrays.asList(
+        new SimpleModule("AvroInputRowParserModule")
```

Comment:
can we call it "AvroExtensionsModule" just to be consistent with name of this extension